packages/scene-composer/src/components/toolbars/floatingToolbar/items/AddObjectMenu.tsx

```diff
@@ -276,10 +277,13 @@ export const AddObjectMenu = (): JSX.Element => {
           uri: modelUri,
           modelType: modelType ?? ext.toUpperCase(),
         };
-
+        const animationComponent: IAnimationComponent = {
```

Comment:
This still adds an animation component to all models even if they can't have an animation.  The gltfComponent is also hold over, it could be any model type not just gltf.